packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -826,13 +826,13 @@ export class Virtualizer<T extends object, V, W> {
     // method to build the final tree.
     let viewsByParentKey = new Map([[null, []]]);
     for (let view of this._children) {
-      if (!viewsByParentKey.has(view.layoutInfo.parentKey)) {
+      if (view.layoutInfo?.parentKey && !viewsByParentKey.has(view.layoutInfo.parentKey)) {
```

Comment:
What's the parent key issue and how do I reproduce it?

Reply:
Hide the first column. Then, quickly show and hide it again. Check console.